Library/Formula/awf.rb

```diff
@@ -0,0 +1,30 @@
+class Awf < Formula
+  desc "awf (A Widget Factory) is a theme preview application for gtk2 and gtk3"
+  homepage "https://github.com/valr/awf"
+  url "https://github.com/valr/awf/archive/v1.3.0.tar.gz"
+  sha256 "911ded8a307beecbb8ee1dab489fa7b1aa9d7965cb34bd938482220505814fa4"
+  head "https://github.com/valr/awf.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "gtk+"
+  depends_on "gtk+3"
+
+  def install
+    inreplace "src/awf.c", "/usr/share", "#{HOMEBREW_PREFIX}/share"
+    system "./autogen.sh"
+    rm "README.md" # lets not have two copies of README
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    assert (bin/"awf-gtk2").exist?
+    assert (bin/"awf-gtk3").exist?
```

Comment:
I'm with Mike on this. Some things just can't practically be tested. Obviously if there's an executable or library we should always enforce testing that, but where neither of those things exist our hands are fairly tied.

If we leave out an existence test here, where do we go in terms of testing in 3/6/9 months time? Unless we have plans & the capability to do automatic GUI testing no better test is going to come up & we should accept these sort of things.